stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx

```diff
@@ -31,13 +31,15 @@ function CountrySiteHeader({
       <section className="header">
         <div className="grid-container fluid">
           <div className="grid-x grid-margin-x align-content-middle">
-            <div className="cell large-9 small-9 align-self-middle top-left">
-              <a href="https://develop.design-system.pages.dev/iframe?id=components-navigation-components-main-navigation-global-header--global-header&globals=locale:english&args=&viewMode=story" className="logo" tabIndex="0">
+            <div className="cell large-9 small-8 align-self-middle top-left">
+              <a href="#" className="logo" tabIndex="0" title="UNDP Logo homepage link">
                 <Logo src={pnud} alt="UNDP Logo" />
               </a>
               <div className="site-title">
-                <span>{siteTitleData.label}</span>
-                <span>{siteTitleData.span}</span>
+                {!!siteTitleData.label && (
+                  <span><a href="#" className="site-title--label" title="UNDP homepage link" aria-hidden="true">{siteTitleData.label}</a></span>
```

Comment:
Hi @Jura, actually, Those classes are remnants of when I was trying to target the "label" and fix some style issues that popped up when I added the links and worked on some of the other header tasks.

Having an empty "label" span in place, even if the "label" didn't exist, was still adding a spacing artifact that became more prominent as I worked on the other tasks. That's why I added the siteTitleData "label" check to remove it if it doesn't exist. 

After that, I went back and forth between using the label and span class names or using the first of/last of combo. I normally just use class names to target, but with the existing first-of already in the css, turning it into a first-of/last-of combo seemed to be smallest code change. It allows the existing css that was focused on the first span (which it assumed to be the "label/Region") to only apply itself if there is both the "label/Region" and "span/Site name". Otherwise it leaves the "span/Site name" alone.

I can remove the first-of/last-of combo, but I will then need to add a class to the title items to target some of the css fixes that popped up with the wrapping links. If I keep the first-of/last-of combo I can remove the class names from the label and sitename. Do you have a preference?

Reply:
@Jura class names are removed and pushed up. Ok to resolve?

Reply:
yup